packages/helpers/src/servers.js

```diff
@@ -5,21 +5,42 @@
    * 
    * return {string} - The server URL.
    */
-module.exports.getServerUrl = (server) => {
+const getServerUrl = (server) => {
   let url = server.host();
 
   //might be that somebody by mistake duplicated protocol info inside the host field
   //we need to make sure host do not hold protocol info
   if (server.protocol() && !url.includes(server.protocol())) {
     url = `${server.protocol()}://${url}`;
   }
-    
+
   if (server.hasPathname()) {
     url = `${url}${server.pathname()}`;
   }
-    
+
   return url;
 };
 
-//TODO: this separate file for helpers for servers represents approach to keep all helpers in separate files related to extractions of data from specific high level AsyncAPI objects. Here we will have more helpers for example related to variables extraction from servers, security, etc.
-    
\ No newline at end of file
+/**
+ * Retrieve a server object from the provided AsyncAPI servers collection.
+ *
+ * @param {Map<string, object>} servers - A map of server names to AsyncAPI server objects.
+ * @param {string} serverName - The name of the server to retrieve.
+ * 
+ * @throws {Error} If the specified server name is not found in the provided servers map.
+ *
+ * @returns {object} The AsyncAPI server object corresponding to the given server name.
+ */
+const getServer = (servers, serverName) => {
+  if (serverName && servers.has(serverName)) {
```

Comment:
servers can also be undefined

Reply:
no, only this

probably `servers = parsedAsyncAPIDocument.servers();` should not be part of `beforeAll`